packages/fluentui/react-northstar/test/specs/components/Menu/MenuItem-test.tsx

```diff
@@ -68,6 +68,15 @@ describe('MenuItem', () => {
     expect(menuItem.text()).toBe('Home');
   });
 
+  it('Wrapper onClick must be called', () => {
+    const onClick = jest.fn();
+    const menuItem = mountWithProviderAndGetComponent(MenuItem, <MenuItem wrapper={{ onClick }}>Home</MenuItem>);
+
+    menuItem.simulate('click');
+
+    expect(onClick).toHaveBeenCalled();
+  });
```

Comment:
```suggestion
  describe('wrapper', () => {
    it('onClick should be called', () => {
      const onClick = jest.fn();
      const wrapper = mountWithProvider(<MenuItem wrapper={{ onClick }}>Home</MenuItem>);

      wrapper.find('MenuItemWrapper').simulate('click');
      expect(onClick).toHaveBeenCalled();
    });
  });
```

Reworked the test a bit